README.md

```diff
@@ -1,66 +1,71 @@
-* 🇭🇳 [Honduras](packages/honduras)
-* 🇭🇰 [Hong Kong](packages/hong-kong)
-* 🇮🇳 [India](packages/india)
-* 🇮🇩 [Indonesia](packages/indonesia)
-* 🇮🇱 [Israel](packages/israel)
-* 🇮🇹 [Italy](packages/italy)
-* 🇯🇵 [Japan](packages/japan)
-* 🇲🇽 [Mexico](packages/mexico)
-* 🇳🇿 [New Zealand](packages/new-zealand)
-* 🇳🇬 [Nigeria](packages/nigeria)
-* 🇵🇰 [Pakistan, Districts](packages/pakistan.districts)
-* 🇷🇴 [Romania](packages/romania)
-* 🇸🇦 [Saudi Arabia](packages/saudi-arabia)
-* 🇰🇷 [South Korea](packages/south-korea)
-* 🇪🇸 [Spain](packages/spain)
-* 🇱🇰 [Sri Lanka](packages/sri-lanka)
-* 🇸🇪 [Sweden](packages/sweden)
-* 🇹🇼 [Taiwan](packages/taiwan)
-* 🇹🇼 [Taiwan, Main Island](packages/taiwan.main)
-* 🇹🇿 [Tanzania](packages/tanzania)
-* 🇹🇭 [Thailand](packages/thailand)
-* 🇹🇳 [Tunisia](packages/tunisia)
-* 🇦🇪 [UAE](packages/uae)
-* 🇺🇦 [Ukraine](packages/ukraine)
-* 🇺🇸 [USA](packages/usa)
-* 🇺🇸 [USA, Counties](packages/usa.counties)
-* 🇺🇸 [USA, States & Territories](packages/usa-.states-territories)
-* 🇺🇸 [USA, Florida](packages/usa.florida)
-* 🇺🇸 [USA, Michigan](packages/usa.michigan)
-* 🇺🇸 [USA, Utah](packages/usa.utah)
-* 🇺🇿 [Uzbekistan](packages/uzbekistan)
-* 🌍 [World](packages/world)
-* 🇿🇼 [Zimbabwe](packages/zimbabwe)
+
+- 🇦🇺 [Australia](packages/australia)
+- 🇦🇹 [Austria](packages/austria)
+- 🇧🇷 [Brazil](packages/brazil)
+- 🇰🇭 [Cambodia](packages/cambodia)
+- 🇨🇲 [Cameroon](packages/cameroon)
+- 🇨🇦 [Canada](packages/canada)
+- 🇨🇻 [Cape Verde](packages/cape-verde)
+- 🇨🇳 [China](packages/china)
+- 🇨🇴 [Colombia](packages/colombia)
+- 🇫🇷 [France, Departments](packages/france.departments)
+- 🇫🇷 [France, Regions](packages/france.regions)
+- 🇩🇪 [Germany](packages/germany)
+- 🇬🇷 [Greece](packages/greece)
+- 🇭🇳 [Honduras](packages/honduras)
+- 🇭🇰 [Hong Kong](packages/hong-kong)
+- 🇮🇳 [India](packages/india)
+- 🇮🇩 [Indonesia](packages/indonesia)
+- 🇮🇷 [Iran](packages/iran)
+- 🇮🇱 [Israel](packages/israel)
+- 🇮🇹 [Italy](packages/italy)
+- 🇯🇵 [Japan](packages/japan)
+- 🇲🇽 [Mexico](packages/mexico)
+- 🇳🇿 [New Zealand](packages/new-zealand)
+- 🇳🇬 [Nigeria](packages/nigeria)
+- 🇵🇰 [Pakistan, Districts](packages/pakistan.districts)
+- 🇷🇴 [Romania](packages/romania)
+- 🇸🇦 [Saudi Arabia](packages/saudi-arabia)
+- 🇰🇷 [South Korea](packages/south-korea)
+- 🇪🇸 [Spain](packages/spain)
+- 🇱🇰 [Sri Lanka](packages/sri-lanka)
+- 🇸🇪 [Sweden](packages/sweden)
+- 🇹🇼 [Taiwan](packages/taiwan)
+- 🇹🇼 [Taiwan, Main Island](packages/taiwan.main)
+- 🇹🇿 [Tanzania](packages/tanzania)
+- 🇹🇭 [Thailand](packages/thailand)
+- 🇹🇳 [Tunisia](packages/tunisia)
+- 🇦🇪 [UAE](packages/uae)
+- 🇺🇦 [Ukraine](packages/ukraine)
+- 🇺🇸 [USA](packages/usa)
+- 🇺🇸 [USA, Counties](packages/usa.counties)
+- 🇺🇸 [USA, States & Territories](packages/usa-.states-territories)
+- 🇺🇸 [USA, Florida](packages/usa.florida)
+- 🇺🇸 [USA, Michigan](packages/usa.michigan)
+- 🇺🇸 [USA, Utah](packages/usa.utah)
+- 🇺🇿 [Uzbekistan](packages/uzbekistan)
+- 🌍 [World](packages/world)
+- 🇿🇼 [Zimbabwe](packages/zimbabwe)
```

Comment:
I guess this is due to autoformatting, but could you add the new map without modifying the rest of the file please?